project/Build.scala

```diff
@@ -49,15 +49,14 @@ object ApplicationBuild extends Build {
   lazy val plugin = Project(appName, base = file("plugin"))
     .settings(baseSettings: _*)
     .settings(
-      resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
-      resolvers += "Typesafe Maven Repository" at "http://repo.typesafe.com/typesafe/maven-releases/",
+      resolvers += "Typesafe Maven Repository" at "https://dl.bintray.com/typesafe/maven-releases/",
       resolvers += "Spy Repository" at "http://files.couchbase.com/maven2",
       resolvers += "Scalaz Bintray Repo"  at "http://dl.bintray.com/scalaz/releases",
-      libraryDependencies += "net.spy" % "spymemcached" % "2.9.0",
+      libraryDependencies += "net.spy" % "spymemcached" % "2.12.0",
```

Comment:
Thanks for your kindness here!
But we are staying at 2.9.0 because there was once a stability issue in a newer version (as I remember).
I won't agree with bumping it to 2.12.0 without stress testing.